lib/mp4/transmuxer.js

```diff
@@ -982,19 +980,17 @@ Transmuxer = function(options) {
     // disassemble MPEG2-TS packets into elementary streams
     pipeline.packetStream
       .pipe(pipeline.parseStream)
-      .pipe(pipeline.elementaryStream);
+      .pipe(pipeline.elementaryStream)
+      .pipe(pipeline.timestampRolloverStream);
 
     // !!THIS ORDER IS IMPORTANT!!
     // demux the streams
     pipeline.elementaryStream
```

Comment:
the three following `pipeline.elementaryStream` should be `pipeline.timestampRolloverStream` so its more clear how the pipeline is structured